contracts/MainframeTokenDistribution.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.18;
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./MainframeToken.sol";
```

Comment:
You could reference `ERC20` here instead to indicate that it doesn't rely on any special functionality of the mainframe token contract, similar to the escrow contract.